File: ParteJava/ProyectoPro3/ProyectoPro3Modelo/src/main/java/pe/edu/pucp/proyectopro3/modelo/auth/Analista.java
package pe.edu.pucp.proyectopro3.modelo.auth;

import pe.edu.pucp.proyectopro3.modelo.crm.TipoDocumento;

public class Analista extends Usuario{
    public Analista(int idUsuario, String nombres, String apellidos, TipoDocumento tipoDocumento, String numeroDocumento, String numeroContacto, String correo, String contrasena) {
        super(idUsuario, nombres, apellidos, tipoDocumento, numeroDocumento, correo, contrasena, numeroContacto);
    }
}

File: ParteC#/ProyectoProgramacion3/ProyectoProgramacion3Model/Model/auth/Analista.cs
using ProyectoProgramacion3Model.Model.crm;
using System;
using System.Collections.Generic;
using System.Text;

namespace ProyectoProgramacion3Model.Model.auth;
public class Analista : Usuario
{
    public Analista()
    {
    }
    public Analista(int idUsuario, string nombres, string apellidos, TipoDocumento tipoDocumento,
        string numeroDocumento, string numeroContacto, string correo, string contrasena) :
        base(idUsuario, nombres, apellidos, tipoDocumento, numeroDocumento, numeroContacto, correo, contrasena)
    {
    }
}
